src/components/NumberIcon/NumberIcon.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import { StoryObj } from '@storybook/react';
+import React from 'react';
+
+import { NumberIcon } from './NumberIcon';
+
+export default {
+  title: 'Atoms/Icons/NumberIcon',
+  component: NumberIcon,
+  args: {
+    'aria-label': '1',
+    number: '1',
+  },
+};
+
+type Args = React.ComponentProps<typeof NumberIcon>;
+
+export const Default: StoryObj<Args> = {};
+
+export const Small: StoryObj<Args> = {
+  args: {
+    size: 'sm',
+  },
+};
+
+export const SuccessLarge: StoryObj<Args> = {
+  args: {
+    size: 'lg',
+    variant: 'success',
+  },
+};
+
+export const SuccessSmall: StoryObj<Args> = {
+  args: {
+    size: 'sm',
+    variant: 'success',
+  },
+};
+
+/**
+ * 1) Disables controls for args that have no affect on this story
+ */
+export const DifferentNumbers: StoryObj<Args> = {
+  /* 1 */
+  argTypes: {
+    number: {
+      table: {
+        disable: true,
+      },
+    },
+    'aria-label': {
+      table: {
+        disable: true,
+      },
+    },
+    'aria-labelledby': {
+      table: {
+        disable: true,
+      },
+    },
+  },
+  render: (args) => (
+    <div>
+      <NumberIcon {...args} aria-label="0" number={0} />
```

Comment:
The `aria-label` values in this story make my wonder if we should just have the number given as the default value.

Reply:
I kept it required without default since I wanted meaningful labels and a simple `1` wouldn't carry meaning?

Reply:
Then I think the examples in storybook and in the component docstring should demonstrate what kind of label we're looking for.

Reply:
Got it, will say `Step 1` and so forth since I imagine that to be the general use case